### PR TITLE
customize visualization dashboard for soa class

### PR DESCRIPTION
**Pull request by Hsuan0923**
- Hide visualization report
- Change the homework title of score report

截圖 2023-10-31 下午10 38 18

### CODE REVIEW

File: application/controllers/app.rb [hunk 1]
```diff
@@ -564,13 +564,17 @@ class App < Roda
               scores.each_key do |key|
                 title[key] = case key
```
Comment (soumyaray) [inline application/controllers/app.rb:565]: Let's discuss this switch statement at our next meeting (please put on Trello!) I'm curious what else `title[key]` is used for other than this switch statemeent.

**Review by soumyaray — APPROVED** [review]

File: application/controllers/app.rb (hunk 1 above)
Comment (Hsuan0923) [reply application/controllers/app.rb:565]: No problem, I'll put it on the Moonbear Trello board!
And should I merge this branch to the master right now or after our meeting on Friday?

Comment (soumyaray) [reply application/controllers/app.rb:565]: Merge into develop for now and you can confirm it after our Friday meeting?

Comment (Hsuan0923) [reply application/controllers/app.rb:565]: OK, done!

**Pull request merged**